0.59.md

```diff
@@ -0,0 +1,348 @@
+
+![source of the `metro.config.js` file in the 0.59 template, demonstrating where to enable `inlineRequires`](resources/inline-requires.png)
+
+We need the community to let us know how it works before we turn it on by default. When you upgrade to 0.59, there will be a new `metro.config.js` file; flip the `inlineRequires` option and give us [your feedback](https://twitter.com/hashtag/inline-requires)! Read more about inline requires [in the performance docs](https://facebook.github.io/react-native/docs/0.56/performance#inline-requires) to see it work and benchmark your app's performance improvement.
+
+## Lean core is underway
+
+React Native is a large and complex project with a complicated repository. This makes the codebase less approachable to contributors, difficult to test, and bloated as a dev dependency. [Lean Core](https://github.com/react-native-community/discussions-and-proposals/issues/6) is our effort to address these issues by migrating code to separate libraries for better management. The past few releases have seen the first steps of this, but [let's get serious](https://www.youtube.com/watch?v=FMLKb4or8yg).
+
+You may notice that **AsyncStorage**, **ImageStore**, **MaskedViewIOS**, **NetInfo**, **Slider**, **Switch**, and **ViewPagerAndroid** are now officially deprecated. This is great news, as there are now owners for these features actively maintaining them. Heed the warning messages and migrate to the new libraries for these features, because in 0.60 they will be removed.
+
+Over the coming months, there will be many more components following this path to a leaner core. We're looking for help with this &mdash; head over to the [lean core umbrella](https://github.com/facebook/react-native/issues/23313) to pitch in.
+
+## CLI improvements
+
+React Native's command like tools are developer's entrypoint to the ecosystem, but they had long-standing issues and lacked official support. The CLI tools have been moved to a separate repository, and a dedicated group of maintainers have already made some exciting improvements.
+
+Logs are formatted much better now. Commands now run nearly instantly &mdash; you'll immediately notice a difference:
+
+![0.58's CLI is slow to start](resources/0.58-cli-speed.png)![0.58's CLI is nearly instananeous](resources/0.59-cli-speed.png)
+
+Some long-standing issues have been fixed too. Thanks to the [dedicated group of maintainers](https://blog.callstack.io/the-react-native-cli-has-a-new-home-79b63838f0e6) working in a [new repository](https://github.com/react-native-community/react-native-cli). This work was proposed in [RFC0003](https://github.com/react-native-community/discussions-and-proposals/blob/master/proposals/0003-Extracted-Command-Line-Tools.md), and we're excited to see it completed.
+
+## Changelog
+
+Without further adieu, the changes are below.
```

Comment:
```suggestion
Without further ado, the changes are below.
```

Apparently -> https://www.eventbuilder.rocks/blog/if-you-say-without-further-adieu-you-really-should-consider-this